### PR TITLE
UHM-4257: Move pih-config files out of Mirebalais Puppet into new "op…

### PR DESCRIPTION
**Pull request by mogoodrich**
…enmrs-config-pih-[country]" repos

### CODE REVIEW

File: configuration/pih/pih-config-default.json [hunk 1]
```diff
@@ -0,0 +1 @@
+pih-config-liberia.json
```
Comment (mogoodrich) [inline configuration/pih/pih-config-default.json]: This file is a just a sym link to provide a default-top-level configuration.   Will have to see if this works on Windows, and if end users find it helpful or confusing.

File: configuration/pih/pih-config-site-default.json [hunk 2]
```diff
@@ -0,0 +1 @@
+pih-config-liberia-pleebo.json
```
Comment (mogoodrich) [inline configuration/pih/pih-config-site-default.json]: This file is a just a sym link to provide a default site configuration

File: configuration/pih/pih-config-default.json (hunk 1 above)
Comment (brandones) [reply configuration/pih/pih-config-default.json]: Do we have to support Windows? Symlinks are not supported on Windows. I do not want to constantly be asking the question "is this copy of these files in sync with this other copy"?

Comment (mogoodrich) [reply configuration/pih/pih-config-default.json]: I don't think we need to support development on Windows (yet?) but these "config" repos need to be at least be updatable by Windows users.

For the "is this file in sync", we are looking to create some sort of watch functionality:
https://pihemr.atlassian.net/browse/UHM-4339

I also recently added this related ticket:
https://pihemr.atlassian.net/browse/UHM-4342

**Pull request merged**